### PR TITLE
Fix  bug when dropout = 0 in GAT

### PR DESCRIPTION
**Pull request by sufeidechabei**
GAT will throw an error when dropout =0, I have fixed it.

### CODE REVIEW

File: examples/pytorch/gat/train.py [hunk 1]
```diff
@@ -93,7 +93,9 @@ def edge_attention(self, edges):
         a = torch.exp(a).clamp(-10, 10)  # use clamp to avoid overflow
         if self.attn_drop:
             a_drop = self.attn_drop(a)
-        return {'a' : a, 'a_drop' : a_drop}
+            return {'a' : a, 'a_drop' : a_drop}
+        return {'a': a, 'a_drop': a }
```
Comment (jermainewang) [inline examples/pytorch/gat/train.py]: Change L95 to `a = self.attn_drop(a)` and use one return statement.

Comment (jermainewang) [reply examples/pytorch/gat/train.py]: Sorry my bad. Use
```python
if self.attn_drop:
    a_drop = self.attn_drop(a)
else:
    a_drop = a
return {'a' : a, 'a_drop' : a_drop}
```
So there is only one return statement.
